exercises/practice/react/lib/react.ex

```diff
@@ -0,0 +1,41 @@
+defmodule React do
+  @doc """
+  Start a reactive system
+  """
+  @spec new(cells :: [%{atom => any}]) :: {:ok, pid}
```

Comment:
The map `%{atom => any}` will be a very specific map, right? One of:
```elixir
%{type: :input, name: String.t(), value: any()}
%{type: :output, name: String.t(), inputs: [String.t()], compute: (...) -> any()}
```
Should we write that in the typespec instead of the generic `%{atom => any}`?